python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-security.j2

```diff
@@ -93,6 +93,12 @@ management security
 {%         if ssl_profile.cipher_list is arista.avd.defined %}
       cipher-list {{ ssl_profile.cipher_list }}
 {%         endif %}
+{%         if ssl_profile.ciphers.v1_0 is arista.avd.defined %}
+      cipher v1.0 {{ ssl_profile.ciphers.v1_0 }}
+{%         endif %}
```

Comment:
```suggestion
{%         if ssl_profile.ciphers.v1_0 is arista.avd.defined %}
      cipher v1.0 {{ ssl_profile.ciphers.v1_0 }}
{%         elif ssl_profile.cipher_list is arista.avd.defined %}
      cipher-list {{ ssl_profile.cipher_list }}
{%         endif %}
```
We can support both key in schema but make the logic to support one in template. But lets discuss before to proceed.
cc: @gmuloc @ClausHolbechArista

Reply:
Looks like the previous key `cipher-list` was completely removed..
```
leaf1(config-s-gmu-mgmt-sec-ssl-profile-BLAH)#cipher ?
  v1.0  The cipher lists for TLS version 1.0, 1.1 and 1.2
  v1.3  The cipher suites for TLS version 1.3

leaf1(config-s-gmu-mgmt-sec-ssl-profile-BLAH)#cipher?
cipher
```

not very backward compatible. so it is either one or the other and never both

Reply:
Okay, updated it!